core/auth/auth.go

```diff
@@ -150,6 +150,19 @@ func KeyAuth(cfg *config.Configuration) (http.RoundTripper, error) {
 		return nil, fmt.Errorf("configuring key authentication: private key could not be found: %w", err)
 	}
 
+	if cfg.TokenCustomUrl == "" {
```

Comment:
Don't we need to also check if `cfg.TokenCustomUrl == nil`?

Reply:
Ah it's a string, I was tricked by the `omitempty`. So if the field is a string and is not defined in a struct the default is `""`?

Reply:
its a string